developments_build/sql/_export.sql

```diff
@@ -130,4 +130,8 @@ FROM EXPORT_devdb;
 DROP TABLE IF EXISTS SHP_housing;
 SELECT * INTO SHP_housing
 FROM SHP_devdb
-WHERE "ResidFlag" = 'Residential';
\ No newline at end of file
+WHERE "ResidFlag" = 'Residential';
+
+DROP TABLE IF EXISTS manual_corrections;
```

Comment:
let's just do a rename in bash, instead of changing the name in the database
```bash
CSV_export housing_input_research
mv housing_input_research.csv manual_corrections.csv
```